Makefile

```diff
@@ -338,11 +338,11 @@ endif
 # PPA includes the required mongodb-server binaries.
 install-snap-dependencies:
 ## install-snap-dependencies: Install the supported snap dependencies
```

Comment:
I wonder if we added a line like:
```
GO_MOD_VERSION=$(shell grep -P "^go \d+\.\d+$" go.mod | cut -d " " -f 2
```

And then that would let us just keep go.mod up to date and not require changing two places.

Reply:
Looks like a much better solution! I have made that suggested change plus the ifeq change required further down. It seems to work will push a new version to this PR. Let me know how that looks.